# Set velocity scaling factor for given planning group

string group_name
float64 value
---
uint8 SUCCEEDED=0
uint8 FAILED=1
uint8 result_status